scss/color.scss

```diff
@@ -0,0 +1,6 @@
+$purple:#484BB4;
+$nav-link: #222222;
+$heading: #333333;
+$title: #7D7D7D;
+$grey-bg:#eaeaea;
```

Comment:
use basic styling, add spaces, use capital case

`$grey-bg: #EAEAEA;`

Reply:
got it